src/screenshot/screenshot-compare.ts

```diff
@@ -134,9 +140,11 @@ async function getMismatchedPixels(pixelmatchModulePath: string, pixelMatchInput
      * The '2500' value that we default to is the value of `jasmine.DEFAULT_TIMEOUT_INTERVAL` (5000) divided by 2.
      */
     const timeout =
-      typeof jasmine !== 'undefined' && jasmine.DEFAULT_TIMEOUT_INTERVAL
-        ? jasmine.DEFAULT_TIMEOUT_INTERVAL * 0.5
-        : 2500;
+      screenshotTimeoutMs !== null
```

Comment:
Can we update the JSDoc for this logic, now that we don't just set the value to 2500?